modules/installation-and-upgrade/pages/container-management/updating-server-containers.adoc

```diff
@@ -1,26 +1,84 @@
 = {productname} Server Upgrade
 
-Before running the upgrade command, it is recommended to upgrade the [literal]``mgradm`` tool first.
+Before running the upgrade command, it is recommended to to update the host operating system.
+Updating the host operating system will also result in the update of the {productname} tooling such as the [literal]``mgradm`` tool.
+
+.Procedure: Upgrading {productname} Server
+
+. Refresh software repositories with [command]``zypper``:
 
-.Procedure: Upgrading Server
-. Refresh software repository with [command]``zypper``:
 +
+
+[source,shell]
 ----
 zypper ref
 ----
-. Apply available updates with [command]``transactional-update``:
+
+. Depending on the host operating system, proceed with these steps:
+
++
+
+ifeval::[{mlm-content} == true]
+For a transactional system such as {sl-micro}: ::
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+For a transactional system such as {leapmicro}: ::
+endif::[]
+
 +
+
+--
+
+Apply available updates with [command]``transactional-update``:
+
+[source,shell]
 ----
 transactional-update
 ----
+
+--
+
++
+
+ifeval::[{mlm-content} == true]
+For {sles}: ::
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+For {leap}: ::
```

Comment:
For now I will not add leap/tumbleweed as it overall mismatch the other parts of the docs. We need this change but we should do that later once we switch from leap mico to Tumbleweed.

For now, only if block for mlm would suffice.